lisa/sut_orchestrator/azure/tools.py

```diff
@@ -62,22 +62,27 @@ def get_generation(self) -> str:
 
 class LisDriver(Tool):
     """
-    This is a virtual tool to detect/install LIS (Linus Integration Services) drivers.
+    This is a virtual tool to detect/install LIS (Linux Integration Services) drivers.
     More info  - https://www.microsoft.com/en-us/download/details.aspx?id=55106
     """
 
     __version_pattern = re.compile(r"^version:[ \t]*([^ \n]*)")
 
+    @property
+    def command(self) -> str:
+        return ""
+
     @property
     def dependencies(self) -> List[Type[Tool]]:
         return [Wget, Modinfo]
 
     @property
     def can_install(self) -> bool:
-        if (
-            isinstance(self.node.os, Redhat)
-            and float(self.node.os.os_version.release) < 7.8
-        ):
+        if len(self.node.os.os_version.release.split(".")) > 2:
```

Comment:
why? 
I will revert changes for this LisDriver, since you will have a PR soon.

Reply:
Because os_version for Redhat or CentOs, we are getting from cat /etc/redhat-release. I think we will get those only with major and minor version

cat /etc/redhat-release
Red Hat Enterprise Linux Server release 6.7 (Santiago)

Reply:
I got exception when float(release) + image OpenLogic CentOS 8_3 latest, at that time the release is 8.3.2011, that's why I made this change.
cat /etc/redhat-release
CentOS Linux release 8.3.2011

Reply:
Got it. I see, yeah can compare with semver.compare(). you can make the change if you want but keep it in a different commit or else, I will push out a new PR. Either way is fine.

Reply:
I'm doubt if all Linux distros follow sematic versions. If we use semver, make sure it fits to all versions. Another way is to implement different comparable version for different distros.